tests/test_ega.py

```diff
@@ -1,7 +1,9 @@
 from argparse import Namespace
+
 from lib.cli import icgc_download_client
```

Comment:
No, the automatic refactoring failed.  I noticed in a while ago, it'll be fixed next commit